.github/workflows/ci.yml

```diff
@@ -85,3 +85,71 @@ jobs:
           zip: all
           manifest-path: test-crate/Cargo.toml
           codesign: '-'
+
+  optional_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
This appears to have been copied from the main test job, but I don't think we need so many matrices to test an option. One for each platform would be fine; CI resources are not infinite.

Reply:
Will remove this after all other comments are been resolved.

Reply:
Update: Removed extra test cases (all comments are fixed, manual testing are done).